theory/particle physics/lepton sources.md

```diff
@@ -7,3 +7,34 @@ nav_order: 2
 ---
 
 ## Lepton Sources
+
+### What are Leptons?
+
+[Leptons](#lepton-sources) are elementary particles, the fundamental building blocks of the universe, alongside quarks. Unlike quarks, leptons don't form larger particles but exist independently, playing a key role in shaping the world around us. There are six types of leptons: three carry a negative charge (electron, muon, and tau), while the other three are neutral (the corresponding neutrinos). Electrons, the most familiar, are integral to the structure of atoms, while [neutrinos](../particle%20physics/neutrinos.html), though nearly invisible to us, flood through space and matter, interacting so rarely that billions pass through your body every second without a trace.
```

Comment:
Can the type of leptons be extracted into a separate heading / paragraph with them in a table or a list?